### ISSUE TITLE
could not install the server on my laptop it's a ryzen 5 5500U

### ISSUE THREAD

**Issue by mokshraj** [Jejkobb/Speechfire#5]
i could not install the server on my laptop .firstly there are no specific instructions for windows , so i had to tinker a bit which o don't really mind but then when i try to install the requirement.txt through pip  it did have it so just had to download the txt myself and i just activated the activate.bat through file manager and than the nvidia librarias don't so i remove them from requirements file and then the tqdm and triton didn't install so i removed those too then, i don't know where this server.py is so i don't know how i activate ,

i use python 3.12 and windows 11

**Comment by sasoder** [comment]
Hi @mokshraj, we haven't tested this on AMD gpus, sorry.

- I updated the readme slightly - you should be able to install all packages with `pip install -r requirements.txt` or `pip install requirements-lock.txt` (the locked versions are confirmed working on my Windows and Linux machine.
- For AMD gpus, you probably have to install something like the ROCm platform, as well as the ROCm PyTorch module, which should allow PyTorch to utilise the gpu you have.
- The `server.py` file is in the root directory, but it probably won't do what you want without the above

Neither of us has access to an AMD gpu, so if you manage to get this working, please let us know what you did and we can update the readme.

**Comment by albinahlback** [comment]
have u tried restarting ur cmputer

**Comment by mokshraj** [comment]
hi @sasoder since windows does not have a  native ROCm can i use a different AMD HIP or directml. with some modifications to the code

**Comment by sasoder** [comment]
Hypothetically it should work - please lmk if you do get it working

**Comment by mokshraj** [comment]
@sasoder one more thing i can't really find the server.py like it's not there i did go c:\users\moksh\speechfire\ but it isn't in there

**Comment by sasoder** [comment]
@mokshraj it's in the root directory of the repo, in the same folder as the requirement files - perhaps you've accidentally deleted it, maybe try recloning the repo

**Comment by mokshraj** [comment]
@sasoder the server runs in the terminal but does not give any out put like the toddled the recording with alt+a and pressed on the youtube seach box then , i just say hello but it doesn't type anything like nothing.

**Comment by sasoder** [comment]
@mokshraj considering the incompatibilities between your gpu and how the transcriptions are being produced, it makes sense for the server itself to start, but the transcriptions not processing correctly - did you install the amd-specific packages/sdk's that are required? Again, I'm unable to test this myself so I can't be of much help

**Comment by mokshraj** [comment]
@sasoder yes I did install the and specific SDK .after which I tested it after restart also but it doesn't work i use firefox by the way.

**Comment by sasoder** [comment]
@mokshraj hmm, try creating a simple python script with the bare minimum code that just takes in a short recording, transcribes and prints the result - if you get that working, you can integrate it into the server script. Definitely recommend chatting with claude or chatgpt to troubleshoot, good luck!
